LICENSE

```diff
@@ -178,15 +178,13 @@
    APPENDIX: How to apply the Apache License to your work.
 
       To apply the Apache License to your work, attach the following
-      boilerplate notice, with the fields enclosed by brackets "[]"
-      replaced with your own identifying information. (Don't include
-      the brackets!)  The text should be enclosed in the appropriate
+      boilerplate notice. The text should be enclosed in the appropriate
       comment syntax for the file format. We also recommend that a
       file or class name and description of purpose be included on the
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright [2021] [Derek Pisner]
+   Copyright 2021 The NiPreps Developers <nipreps@gmail.com>
```

Comment:
I hope this is OK with @dPys.

Reply:
I think as long as I'm listed as a NiPrep developer (I think I am even though not "active"?) and credit would be assigned appropriately for any and all future publications directly related to this repository, then yes, this is absolutely okay with me!

Reply:
Thanks for checking in @dPys. You are listed as a contributor on this repository: https://github.com/nipreps/eddymotion/blob/main/.zenodo.json#L28

Reply:
Perfect. Thanks for confirming (and for helping to maintain this awesome repo!)